Assignment7.txt

```diff
@@ -0,0 +1,19 @@
+Clarke Littlejohn
+
+1) if i recall correctly ++*P moves the pointer then gets the values, *p++ get  the value in the point the increments it
+   and *++p is not an apoerator that works.
+
+2) I am confused by what this si asking it will go by what it is prendence so to will go from right to left or left to right based on the highest order
+
+3) you are working directly with the value so it is always modifing the number.
```

Comment:
Pointers are more efficient in handling arrays and data tables 
They can be used to return multiple values from a function via function arguments 
Pointers permit references to functions 
Pointers allow C to support dynamic memory management 
Pointers reduce length and complexity of programs